packages/xod-client/src/project/components/nodeParts/RegularNodeBody.jsx

```diff
@@ -20,8 +20,9 @@ const RegularNodeBody = props => (
     <rect className="body" {...NODE_BODY_RECT_PROPS} />
     <NodeLabel
       text={props.label || XP.getBaseName(props.type)}
-      width={props.size.width}
+      width={props.size.width - 4}
       height={props.size.height}
+      x={2}
```

Comment:
What are these? Isn’t it left after the exclamation icon?

Reply:
Nope. Moved it into a constant and left comment.
It prevents NodeLabel from sticking to the Node outline.